client/src/app/page.tsx

```
@@ -1,13 +1,17 @@
-import Categories from "@/components/Category/Category";
+import Offer from "@/components/Banner/Offer";
+import WhyChooseUs from "@/components/Banner/WhyChooseUs";
+import PopularCategories from "@/components/Category/Category";
 import Hero from "@/components/Header/Hero";
 import LatestProducts from "@/components/LatestProduct/LatestProduct";
 
 export default function Home() {
   return (
     <div>
       <Hero></Hero>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)